src/Backend/OnboardingInsuranceAPI/Areas/Insurance/InsuranceController.cs

```diff
@@ -0,0 +1,39 @@
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+using OnboardingInsuranceAPI.Extensions;
+
+namespace OnboardingInsuranceAPI.Areas.Insurance;
+
+public class InsuranceController
+{
+    private readonly GetInsurance _handlerGet;
+    private readonly AddInsurance _handlerPut;
+    private readonly ILogger<InsuranceController> _logger;
+    
+    public InsuranceController(GetInsurance handlerGetInsurance, AddInsurance handlerPutInsurance, ILogger<InsuranceController> log)
+    {
+        _logger = log;
+        _handlerGet = handlerGetInsurance;
+        _handlerPut = handlerPutInsurance;
+    }
+
+    [Function("GetInsurance")]
+    public async Task<HttpResponseData> Get([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "insurance/{pid}")] HttpRequestData req, string pid)
+    {
+        //_logger.LogInformation($"C# HTTP trigger function processed a request ReadUserInsurance with pid: {pid}");
```

Comment:
Why leave it commented out?

Reply:
I'll delete it on next branch after implement logging on my handlers

Reply:
I'll delete also logger on constructors DI in controllers